src/js/events/CosmosPackagesActions.js

```diff
@@ -127,9 +129,8 @@ const CosmosPackagesActions = {
       data: JSON.stringify({ packageName, packageVersion }),
       success(response) {
         const cosmosPackage = response.package;
-        if (!cosmosPackage.currentVersion && cosmosPackage.version) {
-          cosmosPackage.currentVersion = cosmosPackage.version;
-        }
+
+        cosmosPackage.currentVersion = cosmosPackage.version;
```

Comment:
This is the first version sent by the response or the version selected in the dropdown.